core/src/main/java/org/mskcc/cbio/portal/dao/DaoCopyNumberSegment.java

```diff
@@ -63,6 +63,19 @@ public static int addCopyNumberSegment(CopyNumberSegment seg) throws DaoExceptio
             return 1;
         }
     }
+
+    public static int addFractionGenomeAltered(int cancerStudyId, int sampleId) throws DaoException {
+        if (!MySQLbulkLoader.isBulkLoad()) {
+            throw new DaoException("You have to turn on MySQLbulkLoader in order to insert fraction genome altered");
+        } else {
+            MySQLbulkLoader.getMySQLbulkLoader("fraction_genome_altered").insertRecord(
+                    Integer.toString(cancerStudyId),
+                    Integer.toString(sampleId),
+                    Double.toString(getCopyNumberActeredFraction(sampleId, cancerStudyId, GlobalProperties.getPatientViewGenomicOverviewCnaCutoff()[0]))
```

Comment:
Should we just hard code this cutoff?

Reply:
I can, if we are going to delete (or not use in the new code) that portal property eventually. Should I also remove the cutoff parameter from the new API?

Reply:
Let's remove that portal property.